docs/guides/file-system-based-automated-bundle-generation.md

```diff
@@ -185,6 +185,14 @@ Once generated, all server entrypoints will be imported into a file named `[Reac
 
 *Note: If specifying separate definitions for client and server rendering, please make sure to delete the generalized `ComponentName.jsx` file.*
 
+### Integrating auto-bundling into CI workflows
+
+Currently, ReactOnRails contains conditional logic that checks for the existence of generated entrypoint files whenever `react_component` or one of its derivative methods are called. This conditional logic is used to determine the relative file path to provide `append_javascript_pack` as an argument.
```

Comment:
This should not be true in production. Production should assume it's built.

The conditional is to generate the pack if needed during development, not to determine the path.

Reply:
You're partially correct. I was wrong that the conditional logic is for determining the path.

To support "legacy" components, I removed that conditional logic & replaced it with conditional logic that just skips adding an `append_javascript_pack` call if requested component doesn't match any generated entrypoint.

That's why `load_pack_for_component` was renamed to `load_pack_for_generated_component`.